<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ripes::CacheTab</class>
 <widget class="QWidget" name="Ripes::CacheTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout_2"/>
       </item>
       <item>
        <widget class="CacheTabWidget" name="cacheTabWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="CacheView" name="cacheView"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CacheTabWidget</class>
   <extends>QWidget</extends>
   <header>cachetabwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CacheView</class>
   <extends>QGraphicsView</extends>
   <header>cachesim/cacheview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
